Makefile

```diff
@@ -30,14 +30,15 @@ endif
 
 WITHOUT_JOBS := $(shell echo $(ALL_MODULES) | tr ' ' '\n' | grep -v -e "nomad" | awk '{print "-target=module." $$0 ""}' | xargs)
 ALL_MODULES_ARGS := $(shell echo $(ALL_MODULES) | tr ' ' '\n' | awk '{print "-target=module." $$0 ""}' | xargs)
+DESTROY_TARGETS := $(shell terraform state list | grep module | cut -d'.' -f1,2 | grep -v -e "fc_envs_disk" -e "buckets" | uniq | awk '{print "-target=" $$0 ""}' | xargs)
 
 # Login for Packer and Docker (uses gcloud user creds)
 # Login for Terraform (uses application default creds)
 .PHONY: login-gcloud
 login-gcloud:
 	gcloud auth login
 	gcloud config set project "$(GCP_PROJECT_ID)"
-	gcloud --quiet auth configure-docker us-central1-docker.pkg.dev
+	gcloud --quiet auth configure-docker "$(GCP_REFION)-docker.pkg.dev"
```

Comment:
There is typo in "REFION" — is the GCP_REGION env var accessible?

Reply:
Thanks! yes, GCP_REGION is